citrix_hypervisor/tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import mock
+import pytest
+
+from datadog_checks.dev.http import MockResponse
+
+from . import common
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield
```

Comment:
do you think we could use the same type of container as amazon_msk which mocks the data to an endpoint to have a form of e2e? ex https://github.com/DataDog/integrations-core/pull/9398